src/ziggurat/config.clj

```diff
@@ -75,6 +75,10 @@
 (defn rabbitmq-config []
   (get (ziggurat-config) :rabbit-mq))
 
+(defn statsd-config []
+  (let [cfg (ziggurat-config)]
+    (get cfg :statsd (:datadog cfg)))) ;; TODO: remove datadog in the future
```

Comment:
can we also log a deprecation notice here?

Reply:
added notice on the README